packages/manager/modules/nutanix/src/service.js

```diff
@@ -305,6 +305,24 @@ export default class NutanixService {
     };
   }
 
+  getDedicatedNodePowerId(nodeId) {
+    return this.$http
+      .get(`/dedicated/server/${nodeId}/boot?bootType=power`)
+      .then(({ data }) => data[0]);
```

Comment:
What about if data is null or empty ?

Reply:
It's not possible. This mean the server has not poweroff status or server not exist